core/src/main/java/org/jdbi/v3/core/statement/Call.java

```diff
@@ -198,5 +203,13 @@ public Object map(CallableStatement stmt) {
                 throw new UnableToExecuteStatementException("Could not get OUT parameter from statement", e, getContext());
             }
         }
+
+        private boolean isNull(CallableStatement stmt) {
```

Comment:
Should this test go into the `OutParamArgument.map` logic instead? Right now it hides all the logic about fetching the value from the statement, and now it is split across two places. Imagine testing `OutParamArgument.map` in isolation; you would expect it to return `null` if the fetched value `wasNull`.

Reply:
i tried a few shapes, this seemed least painful. Otherwise, I need to assign to an object, do null check at the end and return the object or null.